util/generate_man.js

```diff
@@ -91,33 +90,53 @@ const processDovecotMdPost = () => {
 	}
 }
 
+const DovecotMd = (md) => {
+	const parts = md.split(',').map((x) => x.trim())
+	switch (parts[0]) {
+	case 'doveadm':
+		return [{type:'strong', children:[{type:'text', value:'doveadm(1)'}]},
+			{type:'emphasis', children:[{type:'text', value:' ' + parts.slice(1).join(' ')}]}]
```

Comment:
This is hard to read, I suggest we at least explode the object:
```js
return [
  {
    type: 'strong',
    children: [
      { type: 'text', value: 'doveadm(1)' }
    ],
  },
  {
    type: 'emphasis',
    children: [
      { type: 'text', value: ` ${parts.slice(1).join(' ')}` }
    ]
  }
]
```

Same with the `case 'setting'`, the others are ok I guess.